class.jetpack-gutenberg.php

```diff
@@ -384,6 +390,19 @@ public static function get_availability() {
 		return array_merge( $available_extensions, $unwhitelisted_blocks, $unwhitelisted_extensions );
 	}
 
+	/**
+	 * Check if an extension/block is already registered
+	 *
+	 * @since 7.2
+	 *
+	 * @param string $extension Name of extension to check.
+	 *
+	 * @return bool
+	 */
+	public static function is_registered( $extension ) {
```

Comment:
Should we use `$slug` here instead of `$extensions` for consistency?

https://github.com/Automattic/jetpack/blob/9d7f20797168a2c62770f34b2e5f3f22a813fd5d/class.jetpack-gutenberg.php#L21